linkerd/app/outbound/src/lib.rs

```diff
@@ -420,7 +420,7 @@ impl Config {
                 // Eagerly fail requests when the proxy is out of capacity for a
                 // dispatch_timeout.
                 .push_failfast(dispatch_timeout)
-                .push(metrics.http_errors)
+                // .push(metrics.http_errors)
```

Comment:
Is this another bound possibly missing?

Reply:
This doesn't work for unrelated reasons, and is updated on a separate branch; I had to comment it out here because we were not previously actually pushing the `http_admit_request` stack.